OrbitGl/App.cpp

```diff
@@ -805,15 +805,27 @@ void OrbitApp::LoadModules(int32_t process_id,
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadModulesFromPreset(const std::shared_ptr<Process>& process,
                                      const std::shared_ptr<Preset>& preset) {
-  std::vector<std::shared_ptr<Module>> modules;
+  std::vector<std::shared_ptr<Module>> modules_to_load;
   for (const auto& pair : preset->m_Modules) {
     const std::string& module_path = pair.first;
     const auto& module = process->GetModuleFromPath(module_path);
-    if (module != nullptr && !module->IsLoaded()) {
-      modules.emplace_back(std::move(module));
+    if (module == nullptr) {
+      SendErrorToUi(
+          "Preset loading incomplete",
+          absl::StrFormat("The module \"%s\" is not loaded by process \"%s\".",
+                          module_path, process->GetName()));
+      continue;
     }
+    if (module->IsLoaded()) {
+      CHECK(module->m_Pdb != nullptr);
```

Comment:
Why is this a CHECK and not an error case that we must handle?

Reply:
Because a loaded module always has a pdb. Thats kind of the definition if a module being loaded. In an easier world maybe `IsLoaded` would return if the module has a pdb, instead of having a separate bool flag. But also, pdb in general is a windows term and currently a very shallow layer and one of the things we are trying to replace with newer data models

Reply:
Ok, sounds good.